keps/sig-apps/2214-array-job/kep.yaml

```diff
@@ -0,0 +1,42 @@
+title: Array Job
+kep-number: 2214
+authors:
+  - "@alculquicondor"
+owning-sig: sig-apps
+status: provisional
+creation-date: 2020-12-29
+reviewers:
+  - "@soltysh"
+  - "@kow3ns"
+  - "@erictune@"
+  - "@ahg"
+approvers:
+  - "@janetkuo"
+  - "@kow3ns"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.21"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.21"
+  beta: "v1.22"
+  stable: "v1.23"
```

Comment:
I think the graduation plan requires being in beta for two releases, so stable will be 1.24 at the earliest.